app/api/routers/breeze_buddy/templates/handlers.py

```
@@ -612,3 +636,206 @@ async def delete_template_handler(
+        # order; page 1 already holds it, later pages fetch just the head.
+        if page == 1 and versions:
+            active_version = versions[0].version_number
+        else:
+            head, _ = await list_template_versions(template_id, 1, 0)
+            active_version = head[0].version_number if head else None
+        return TemplateVersionListResponse(
+            versions=versions, total=total, active_version=active_version
+        )
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)